src/libponyc/codegen/genfun.c

```diff
@@ -186,6 +186,14 @@ static void make_prototype(compile_t* c, reach_type_t* t,
     make_function_debug(c, t, m, m->func);
   }
 
+    // If not partial, add nounwind.
+    if(ast_id(ast_childidx(m->r_fun, 5)) != TK_QUESTION)
+    {
+      LLVMAddFunctionAttr(m->func, LLVMNoUnwindAttribute);
+      if(handler)
+        LLVMAddFunctionAttr(m->func_handler, LLVMNoUnwindAttribute);
+    }
```

Comment:
Seems to be an indentation issue here - these lines should have two fewer spaces of indentation, to match the blocks above.